cloudflare/src/framework/apiclient.rs

```diff
@@ -17,3 +18,15 @@ pub trait ApiClient {
         QueryType: Serialize,
         BodyType: Serialize;
 }
+
+#[async_trait]
+pub trait AsyncApiClient {
```

Comment:
Self-comment: this was the previous async_api::ApiClient , which I renamed here to disambiguate with the sync client

Reply:
Let's not move symbols into new modules -- it's a breaking change and doesn't seem necessary. Same with all the other moves. Ideally this PR would not cause any breaking changes (i.e. would not move any symbols out of their current modules). Adding new symbols is fine though.

Reply:
I've reverted back the breaking changes, resulting in a smaller diff. So that's good news.

The downside is that some of the changes did make sense, i.e., things do seem a little out of place, but I guess we can live with that. Furthermore, this requires spreading around some random `#[cfg(feature = "reqwest-http")]` which is not great, but it's the cost of backwards compatibility.